### DISCUSSION TITLE
DB download error

### DISCUSSION THREAD

**Discussion by marsty339** [aquasecurity/trivy#6825] [Q&A]
### Question

trivy scan error unacceptable mediay type application/vnd.aquasec.trivy.db.layer.v1.tar+gzip
I run command:
oras pull ghcr.io/aquasecurity/trivy-java-db:1
 
oras push --export-manifest manifest.json xx.xx.xx.xx/aquasec/trivy-java-db:1 db.tar.gz
 
jq '.layers[0].mediaType="application/vnd.aquasec.trivy.db.layer.v1.tar+gzip"' manifest.json > new_manifest.json
 
oras manifest push xx.xx.xx.xx/aquasec/trivy-java-db:1 new_manifest.json

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

**Comment by DmitriyLewen** [comment]
Hello @marsty339 
Thanks for your interest to Trivy!

You need to use `2` schema version (image tag):
```bash
oras pull ghcr.io/aquasecurity/trivy-db:2
```
https://aquasecurity.github.io/trivy/v0.52/docs/advanced/air-gap/#download-the-vulnerability-database

Regards, Dmitriy
